### PR TITLE
Remplace render for fetch.

### PR DESCRIPTION
**Pull request by PrestaSafe**
For making file overridable by any template.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/ps_facetedsearch/134)

### CODE REVIEW

File: src/Product/SearchProvider.php [hunk 1]
```diff
@@ -202,7 +202,7 @@ public function renderFacets(ProductSearchContext $context, ProductSearchResult
             return '';
         }
 
-        return $this->module->render(
+        return $this->module->fetch(
             'views/templates/front/catalog/facets.tpl',
```
Comment (PierreRambaud) [inline src/Product/SearchProvider.php]: Why not `module:ps_facetedsearch/` here?

**Comment by PierreRambaud** [comment]
Thanks for your contribution.
Phpstan is triggering an error:

```
 19/19 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100% ------ ----------------------------------------------------------------------- 
  Line   /var/www/html/modules/ps_facetedsearch/src/Product/SearchProvider.php  
 ------ ----------------------------------------------------------------------- 
  205    Parameter #2 $cache_id of method ModuleCore::fetch() expects null,     
         array<string, mixed> given.                                            
  236    Parameter #2 $cache_id of method ModuleCore::fetch() expects null,     
         array<string, mixed> given.   
```

File: src/Product/SearchProvider.php (hunk 1 above)
Comment (PrestaSafe) [reply src/Product/SearchProvider.php]: Hum this IS strange. I've replaced it. It need on Évry fetch

**Comment by PierreRambaud** [comment]
Because it wasn't working and I wasn't able to push on your branch, I create a new PR.
See my commit message for explanations.
https://github.com/PrestaShop/ps_facetedsearch/pull/135

(I keep your commit in my PR :wink: Thanks agains)

**Comment by matks** [comment]
Thank you @PrestaSafe

**Pull request closed without merging**